html/block and inline.html

```diff
@@ -1,16 +1,21 @@
 <html>
+
 <body>
 
-<div style="background-color:rgb(3, 85, 81);color:white;padding:20px;">
-  <h2>London</h2>
-  <p>London is the capital city of England. It is the most populous city in the United Kingdom, with a metropolitan area of over 13 million inhabitants.</p>
-  <p>Standing on the River Thames, London has been a major settlement for two millennia, its history going back to its founding by the Romans, who named it Londinium.</p>
-</div> 
+  <div style="background-color:rgb(3, 85, 81);color:white;padding:20px;">
```

Comment:
I've already told you to move this style rules in style section not here in style atribute.